src/include/buffer.h

```diff
@@ -515,12 +515,18 @@ namespace buffer CEPH_BUFFER_API {
       friend class list;
 
     public:
+      contiguous_appender(char* pos)
+        : pbl(nullptr),
```

Comment:
@rzarzynski NIT please consider to keep title of PR within 50 characters.
https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#title-of-pull-requests-and-title-of-commits